packages/@okta/vuepress-site/docs/concepts/interaction-code/index.md

```diff
@@ -0,0 +1,91 @@
+
+## The Interaction Code flow
+
+The Interaction Code flow is similar to the [OAuth 2.0 Authorization Code flow with PKCE](/docs/concepts/oauth-openid/#authorization-code-flow-with-pkce). All clients are required to pass along a client ID, as well as a Proof Key for Code Exchange (PKCE), to keep the flow secure. Confidential clients such as web apps must also pass a client secret in their authorization request. The user can start the authorization request with minimal information, relying on the client to facilitate the interactions with the Okta Authorization Server to progressively authenticate the user. The series of interactions, which could include multifactor authentication steps, is secured using the `interaction_handle`. After successfully completing the remedial interactions, the client receives an `interaction_code` that they can then redeem for tokens with the [`/token`](/docs/reference/api/oidc/#token) endpoint.
+
+The following table describes the parameters introduced for the Interaction Code grant type flow:
+
+| Interaction Code grant parameter           | Description   |
+| --------------------------------           | -----------   |
+| `interaction_handle` | The `interaction_handle` is an opaque, immutable value that is provided by the Okta Authorization Server. The client can use the `interaction_handle` to interact with the Okta Authorization Server directly. The client is responsible for saving the `interaction_handle` and using it for the duration of the transaction. Any public/confidential client that is configured to use the Interaction Code grant type can obtain an `interaction_handle`. If all the remediation steps are successfully performed, an `interaction_code` is returned as part of the success response.            |
+| `interaction_code` |  The `interaction_code` is a one-time use, opaque code that the client can exchange for tokens using the Interaction Code grant type. This code enables a client to redeem a completed Identity Engine interaction for tokens without needing access to an Authorization Server’s session. |
+
+
+The following sequence of steps is a typical Interaction Code flow with the Identity Engine:
+
+<!--
+See http://www.plantuml.com/plantuml/uml/
+
+@@startuml
+skinparam monochrome true
+actor "Resource Owner (User)" as user
+participant "Client" as client
+participant "Authorization Server (Okta)" as okta
+participant "Resource Server (Your App)" as app
+
+user -> client: Start auth with user info
+client -> client: Generate PKCE code verifier & challenge
+client -> okta: Authorization request w/ code_challenge, scopes, and user info
+okta -> client: Sends interaction_handle in 200 error response (for required interaction)
+okta <-> client: Remedial interaction w/ interaction_handle
+client <-> user: Remedial interaction
```

Comment:
As discussed offline, we're not really representing that we're transitioning from the OAuth `/interact` API to the IdX/Identity Engine API for remediation. Which may be ok as the IdX API isn't doc'd.

In terms of remedial interaction, this last line isn't representing that user data is being relayed back to Okta, i.e user->client->okta